src/main.ts

```diff
@@ -13,6 +13,8 @@ async function bootstrap() {
     }),
   );
 
+  app.enableShutdownHooks();
```

Comment:
hmm, I'm wondering if we currently need it - does the application crash without it on shutdown?

Reply:
if we don't need it then we should remove it. Especially as it consumes resources: https://docs.nestjs.com/fundamentals/lifecycle-events